manual/src/library/Makefile

```diff
@@ -1,7 +1,10 @@
 ROOTDIR = ../../..
+include $(ROOTDIR)/Makefile.common
+LD_PATH = $(ROOTDIR)/otherlibs/str $(ROOTDIR)/otherlibs/unix
 
-TEXQUOTE = $(ROOTDIR)/runtime/ocamlrun ../../tools/texquote2
-CAMLLATEX = CAML_LD_LIBRARY_PATH="" $(OCAMLRUN) $(addprefix -I ,$(LD_PATH)) \
+TOOLS = ../../tools
```

Comment:
This definition doesn't match the one in 5.1 . May I request that we keep the same Makefile across version (except for the `ocamltex` renaming)?

Reply:
Yes, synchronizing with the 5.1 Makefile is a very good idea. I have updated the changes to the PR. Thank you!